src/main/java/core/basesyntax/strategy/transactions/impl/AdderProducer.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.strategy.transactions.impl;
+
+import core.basesyntax.db.StorageDao;
+import core.basesyntax.strategy.transactions.FruitTransaction;
+import core.basesyntax.strategy.transactions.TransactionProducer;
+
+public class AdderProducer implements TransactionProducer {
+    @Override
+    public void transactionProduce(FruitTransaction fruitTransaction,
+                             StorageDao storageDao) {

```

Comment:
`StorageDao` looks more like dependency and not like a parameter, 
let's create additional field and set it through constructor